risethesis.cls

```diff
@@ -546,68 +546,29 @@ requirements for the degree of \@degree\ in \@majorfielden.
 \def\presentationpage{%
   \if@openright\cleardoublepage\else\clearpage\fi
   \thispagestyle{empty}
-  
-	\begin{figure}[ht]
-	\begin{minipage}[b]{1.5cm}
-	  \includegraphics[width=1.5cm]{ufpelogo}
-	\end{minipage}
-	\hspace{0.2cm}
-	\begin{minipage}[b]{16cm}
-	  {
-		  \if\@language0
-    		  \sf\normalsize
-    		  \@universitypt
-    		  \vskip 0.1cm
-    		  \if\@showinstitute1
-    		    \@institutept
```

Comment:
Blz. Era só pra confirmar mesmo. vou esperar, então. valeu!
On Jun 1, 2016 6:03 PM, "Moises Rodrigues" <notifications@github.com> wrote:

> In risethesis.cls
> <https://github.com/yguarata/risethesis/pull/4#discussion_r65442091>:
>
> > @@ -546,68 +546,29 @@ requirements for the degree of \@degree\ in \@majorfielden.
> >  \def\presentationpage{%
> >    \if@openright\cleardoublepage\else\clearpage\fi
> >    \thispagestyle{empty}
> > -
> > -	\begin{figure}[ht]
> > -	\begin{minipage}[b]{1.5cm}
> > -	  \includegraphics[width=1.5cm]{ufpelogo}
> > -	\end{minipage}
> > -	\hspace{0.2cm}
> > -	\begin{minipage}[b]{16cm}
> > -	  {
> > -		  \if\@language0
> > -    		  \sf\normalsize
> > -    		  \@universitypt
> > -    		  \vskip 0.1cm
> > -    		  \if\@showinstitute1
> > -    		    \@institutept
>
> Cara, foi o que eu entendi aqui. Um colega aqui do lab acabou de falar com
> a bibliotecária do CCEN e ela falou pra tirar esses campos da folha de
> rosto, colocar o nome do autor em cima e mover o do orientador pra
> esquerda. Ele imprimiu e entregou tudo dessa maneira. Ele usou o teu
> template e a moça pediu explicitamente pra mudar isso aí. Estou no processo
> de entrega da minha tese agora e devo ir lá amanhã ver isso. Vou mostrar a
> nova versão e pergunto pra ela novamente se é assim mesmo. Segura o pull
> request até amanhã que vou lá e pergunto novamente para termos certeza
> absoluta.
> … <#m_-8740078379171379786_>
> On Wed, Jun 1, 2016 at 4:38 PM, yguarata ***@***.***> wrote: In
> risethesis.cls <#4 (comment)
> <https://github.com/yguarata/risethesis/pull/4#discussion_r65427835>>: >
> @@ -546,68 +546,29 @@ requirements for the degree of ***@***.***\ in
> ***@***.*** > \def\presentationpage{% >
> ***@***.***\cleardoublepage\else\clearpage\fi > \thispagestyle{empty} > - >
> - \begin{figure}[ht] > - \begin{minipage}[b]{1.5cm} > -
> \includegraphics[width=1.5cm]{ufpelogo} > - \end{minipage} > -
> \hspace{0.2cm} > - \begin{minipage}[b]{16cm} > - { > - ***@***.*** > -
> \sf\normalsize > - ***@***.*** > - \vskip 0.1cm > - ***@***.*** > -
> ***@***.*** O nome do instituto e do departamento foram removidos nessa
> alteração. É isso mesmo? — You are receiving this because you authored the
> thread. Reply to this email directly, view it on GitHub <
> https://github.com/yguarata/risethesis/pull/4/files/75790a696788672d2c05e48c87b716b1c663c0ab#r65427835>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ABQ6ghTeN67u1Cfz1bYXjQ0CM2nCZFcaks5qHd-sgaJpZM4IrjW->
> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/yguarata/risethesis/pull/4/files/75790a696788672d2c05e48c87b716b1c663c0ab#r65442091>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAxAXTf765O9tK9P3Bm1sy4ZLfmbzByoks5qHfOcgaJpZM4IrjW->
> .
>